### PR TITLE
CLI should stop creating, updating and deleting service accounts

### PR DESCRIPTION
**Pull request by i14h**
### Description

To use the new managed service accounts on Extensions backend, this commit removes code creating, modifying and deleting ServiceAccounts on `ext:install`, `ext:update` and `ext:uninstall`. In addition, it removes `rolesHelper.ts` which is no longer needed. 

This should be released only after the Firebase Console has fully ramped up the experiment to do the same

### Scenarios Tested

`npm test`

### CODE REVIEW

File: src/commands/ext-update.ts [hunk 1]
```diff
@@ -203,7 +203,6 @@ export default new Command("ext:update <extensionInstanceId> [localDirectoryOrUr
         source: newSource,
         rolesToAdd: _.get(newSpec, "roles", []),
```
Comment (laurenzlong) [inline src/commands/ext-update.ts:204]: `rolesToAdd` and `rolesToRemove` are both unnecessary now, and `update` method needs to be revised to no longer have them as parameters

File: src/extensions/updateHelper.ts [hunk 2]
```diff
@@ -224,7 +223,6 @@ export interface UpdateOptions {
   params?: { [key: string]: string };
   rolesToAdd: extensionsApi.Role[];
```
Comment (laurenzlong) [inline src/extensions/updateHelper.ts:224]: rolesToAdd and rolesToRemove are no longer necessary

File: src/commands/ext-update.ts (hunk 1 above)
Comment (laurenzlong) [reply src/commands/ext-update.ts:204]: lines 194 to 198 also need to be removed.

File: src/commands/ext-uninstall.ts [hunk 3]
```diff
@@ -35,9 +34,6 @@ export default new Command("ext:uninstall <extensionInstanceId>")
       throw err;
     }
     if (!options.force) {
-      const serviceAccountMessage = `Uninstalling deletes the service account used by this extension instance:\n${clc.bold(
```
[the remark this answers is not in the document]
Comment (laurenzlong) [reply src/commands/ext-uninstall.ts:38]: Good point, we should show the message still, and this is consistent with the Console experience, where we tell users about their resources and service account being deleted.

File: src/commands/ext-uninstall.ts [hunk 4]
```diff
@@ -91,28 +86,6 @@ export default new Command("ext:uninstall <extensionInstanceId>")
         logPrefix
       )}: deleting your extension instance's service account.`;
```
Comment (joehan) [inline src/commands/ext-uninstall.ts:87]: This should be deleted

**Comment by i14h** [comment]
@laurenzlong @joehan updated

**Comment by laurenzlong** [comment]
Approved, please update with the latest changes from "master" and use "Squash and merge" option when merging the Pull Request. Thanks!

**Comment by i14h** [comment]
@laurenzlong @agau4779 rebased with master and added line to CHANGELOG.md

**Pull request merged**